lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -40,8 +40,8 @@ mapping:
         required: false
         desc: |
           If running behind a proxy server and Galaxy is served from a subdirectory,
-          enable the proxy-prefix filter and set the prefix in the
-          [filter:proxy-prefix] section above.
+          enable the proxy-prefix filter, uncomment the 'mount: /galaxy=galaxy.webapps.galaxy.buildapp:uwsgi_app()'
```

Comment:
Probably also need to drop `enable the proxy-prefix filter` here?

Reply:
@natefoo does this still stand? ( `#filter-with: proxy-prefix` is still relevant, no?)

Reply:
Not with uWSGI